src/js/helpers/offsetCalculator.js

```diff
@@ -10,9 +10,18 @@
 
 import getOffset from './../libs/offset';
 import getInlineOption from './getInlineOption';
+import validContainer from './container';
 
-export const getPositionIn = (el, defaultOffset, defaultAnchorPlacement) => {
-  const windowHeight = window.innerHeight;
+export const getPositionIn = (
+  el,
+  defaultOffset,
+  defaultAnchorPlacement,
+  container
```

Comment:
I think the container should be the second argument here. It’s more important than offset or offsetAnchorPlacement